src/utils/with_options/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "with_options"
+version = { workspace = true }
+edition = { workspace = true }
+homepage = { workspace = true }
+keywords = { workspace = true }
+license = { workspace = true }
+repository = { workspace = true }
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+proc-macro2 = "1"
+quote = "1"
+serde = "1"
+serde_yaml = "0.9"
+syn = { version = "2", features = ["full"] }
+walkdir = "2"
+
+[lib]
+proc-macro = true
+
+[target.'cfg(not(madsim))'.dependencies]
+workspace-hack = { path = "../../workspace-hack" }
```

Comment:
```suggestion
```

Seems unnecessary for proc-macro crates? cc @xxchan

Reply:
Yes, we should not do it.
https://github.com/risingwavelabs/risingwave/pull/12961